Source/Core/Core/HW/WiiSave.cpp

```diff
@@ -143,7 +143,7 @@ class NandStorage final : public Storage
       else if (file.type == SaveFile::Type::Directory)
       {
         const FS::Result<FS::Metadata> meta = m_fs->GetMetadata(*m_uid, *m_gid, path);
-        if (!meta || meta->is_file)
+        if (meta && meta->is_file)
```

Comment:
@leoetlino I'm pretty sure this was an accidentally flipped condition, as it would fail if the directory didn't exist, and only create it if it already exists. Want your input though.

Reply:
Oh whoops, this is indeed supposed to fail only if the file already exists and is not a directory (and it should just skip creating the directory if it already exists), nice catch!

Reply:
Thanks. I figured it was backwards, since it didn't make any sense. Just wanted to be sure there wasn't some unorthodox reason it was like this.